content/api/sending-domains.apib

```diff
@@ -410,346 +436,64 @@ Delete an existing sending domain.
-
-The domain's `status` object is returned on success.
+List an overview of all sending domains. By default, all domains are returned. Use the query parameters to filter on the various status options. 
 
 + Parameters
-    + domain (required, string, `example1.com`) ... Name of the domain
-
-+ Request Verify DKIM (application/json)
-
-    + Headers
-
-            Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
-    + Body
-
-           {
-               "dkim_verify": true
-           }
-
-
-+ Response 200 (application/json)
-
-        {
-            "results": {
-                "ownership_verified": true,
-                "dns": {
-                    "dkim_record": "k=rsa; h=sha256; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQC+W6scd3XWwvC/hPRksfDYFi3ztgyS9OSqnnjtNQeDdTSD1DRx/xFar2wjmzxp2+SnJ5pspaF77VZveN3P/HVmXZVghr3asoV9WBx/uW1nDIUxU35L4juXiTwsMAbgMyh3NqIKTNKyMDy4P8vpEhtH1iv/BrwMdBjHDVCycB8WnwIDAQAB"
-                },
-                "dkim_status": "valid",
-                "cname_status": "unverified",
-                "mx_status": "unverified",
-                "compliance_status": "pending",
-                "spf_status": "unverified",
-                "abuse_at_status": "unverified",
-                "postmaster_at_status": "unverified",
-                "verification_mailbox_status": "unverified"
-            }
-        }
-
-+ Request Verify CNAME (application/json)
-
-    + Headers
-
-            Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
-    + Body
-
-            {
-               "cname_verify": true
-            }
-
+    + ownership_verified (optional, boolean, `true`) ... Ownership verified flag. Valid values are `true` or `false`.
+    + dkim_status (optional, enum) ... DKIM status filter. <br />Possible Values:  `valid`, `invalid`, `unverified`, `pending`.
+    + cname_status (optional, string) ... CNAME status filter. <br />Possible Values: `valid`, `invalid`, `unverified`, `pending`.
+    + mx_status (optional, string) ... MX status filter. <br />Possible Values: `valid`, `invalid`, `unverified`, `pending`.
+    + abuse_at_status (optional, string) ... abuse@ status filter. <br />Possible Values: `valid`, `invalid`, `unverified`, `pending`.
+    + postmaster_at_status (optional, string) ... postmaster@ status filter. <br />Possible Values: `valid`, `invalid`, `unverified`, `pending`.
+    + compliance_status (optional, string) ... compliance status filter. <br />Possible Values: `valid`, `invalid`, `unverified`, `pending`.
+    + is_default_bounce_domain (optional, boolean) ... Is default bounce domain flag. Valid values are `true` or `false`.
 
```

Comment:
None of these parameters are coming out as their actual type, all strings. Is that a rule?

Reply:
Bug in the renderer/parser. It is on the backlog to tackle.

Reply:
Nope, just a mistake. Fixing.